src/data/store/WfsFeatures.js

```diff
@@ -218,6 +212,32 @@ Ext.define('GeoExt.data.store.WfsFeatures', {
         return totalCount;
     },
 
+    /**
+     * Sends the sortBy parameter to the WFS Server
+     * If multiple sorters are specified then multiple fields are
+     * sent to the server.
+     * Ascending sorts appends +A and descending sorts +D
+     * E.g. sortBy=attribute1+D,attribute1+A
+     * @private
+     * @return {String} The sortBy string
+     */
+    createSortByParameter: function() {
+
+        var me = this;
+        var sortStrings = [];
+        var direction;
+        var property;
+
+        Ext.each(me.getSorters().items, function(sorter) {

```

Comment:
I am not sure if `me.getSorters().items` is reliable here since `items` is not a [public property of Ext.util.SorterCollection](https://docs.sencha.com/extjs/6.2.0/classic/Ext.util.SorterCollection.html#properties).  Maybe
```suggestion
        me.getSorters().each(function(sorter) {
```
might be the more robust approach. Altough knowing that `.items` is alive for around 10 years and won't disappear.

Reply:
Yes much better. `.items` felt clunky.